src/Corporation/ui/MaterialElem.tsx

```diff
@@ -68,7 +67,7 @@ export function MaterialElem(props: IMaterialProps): React.ReactElement {
     if (isString(mat.desiredSellAmount)) {
       sellButtonText = (
         <>
-          Sell ({formatBigNumber(mat.actualSellAmount)}/{mat.desiredSellAmount[1]})
+          Sell ({formatBigNumber(mat.actualSellAmount)}/{mat.desiredSellAmount})
```

Comment:
Wtf, was this just showing nan?

Reply:
no it was showing the secoind string of the string
eg when input was MAX it shows A , PROD showed R

Reply:
Oh of course. Still bad though